tests/lib/prepare-restore.sh

```diff
@@ -675,6 +680,32 @@ restore_suite_each() {
         "$TESTSTOOLS"/cleanup-state pre-invariant
     fi
     tests.invariant check
+
+    if os.query is-focal || os.query is.fedora; then
+        set +x
+        pkill inotifywait
+        local created_files=$( sudo cat /tmp/fs.output | grep " CREATE " | awk '{ print $1$3 }' )
```

Comment:
Could use a comment about the format of the logs, eg:
```
# the output file contains lines:
# /etc/ CREATE foo          # created file /etc/foo
# /etc/ CREATE,ISDIR foo   # created directory /etc/foo
```

Reply:
Sure, thanks for the suggestion